WalletWasabi.Fluent/ViewModels/Wallets/Settings/ExcludedCoinsViewModel.cs

```diff
@@ -45,9 +45,24 @@ protected override void OnNavigatedTo(bool isInHistory, CompositeDisposable disp
 
 		CoinList.Selection
 			.ToObservableChangeSet()
-			.ToCollection()
-			.Throttle(TimeSpan.FromMilliseconds(100), RxApp.MainThreadScheduler)
-			.DoAsync(async x => await _wallet.Coins.UpdateExcludedCoinsFromCoinjoinAsync(x.ToArray()))
+			.Buffer(TimeSpan.FromMilliseconds(10), RxApp.MainThreadScheduler)
```

Comment:
10 ms here because otherwise when you change a group it feels laggy. With 10 ms it's better. Buffer is also required to handle the change in groups